torchgen/gen_lazy_tensor.py

```diff
@@ -281,7 +285,10 @@ def run_gen_lazy_tensor(
     lazy_value_class: str = "torch::lazy::Value",
     lazy_tensor_ptr: str = "LazyTensorPtr",
 ) -> None:
-
+    lv_tokens = lazy_value_class.split("::")
```

Comment:
I'm actually thinking we shouldn't be using `lazy_value_class` directly anywhere else.
If we need the cpp type we should always do `valueT.cpp_type()`.

Reply:
i agree that lazy_value_class should be consumed only by your setter, then everything else should use the getter.

but I think i prefer passing lazy_value_class from this file along with all the other args, just so the list of args is 1:1 everywhere and uncomplicated.  (remember, we do want to clean up all those args somehow, but i'm not sure if putting a new 'setter' for each one at this level would be cleaner in general).  Wdyt about hiding the 'setter' call inside the GenLazy.. constructor?  (oh yea, there isn't a constructor.  can it have one?  if not.. i guess i'm ok with this).